apps/studio/components/to-be-cleaned/Storage/StorageExplorer/ColumnContextMenu.tsx

```diff
@@ -60,15 +60,13 @@ const ColumnContextMenu = ({ id = '' }: ColumnContextMenuProps) => {
 
   return (
     <Menu id={id} animation="fade">
-      {canUpdateFiles && (
-        <>
-          <Item onClick={({ props }) => onSelectCreateFolder(props.index)}>
-            <FolderPlus size="14" strokeWidth={1} />
-            <span className="ml-2 text-xs">New folder</span>
-          </Item>
-          <Separator />
-        </>
-      )}
+      {canUpdateFiles && [
+        <Item key="create-folder" onClick={({ props }) => onSelectCreateFolder(props.index)}>
```

Comment:
Does this really fixes https://supabase.sentry.io/issues/5998687405?project=5459134?

Reply:
yup! this is a slight oddity to conditionally render react-contextify `Items` where they must be rendered as an array and not within a fragment - i'm not sure why this is the case though, but we did do something similar [here](https://github.com/supabase/supabase/blob/master/apps/studio/components/to-be-cleaned/Storage/StorageExplorer/ItemContextMenu.tsx#L86) as well

Reply:
you can verify this bug if you check local `master` atm - right click in the storage explorer on a column and select new folder, a UI error will show up

Reply:
Yup, you're right. Checked on prod and preview. LGTM.